web/apps/labelstudio/src/pages/Projects/Projects.scss

```
@@ -13,6 +13,8 @@
   &__list {
     padding: 40px;
     display: grid;
+
+    // review in the pr this will fail in CI
     grid-autoflow: columns;
```

im sure the initial intent was this
```suggestion
    grid-auto-flow: column;
```

---

This is here since 21, I think we can remove the line.